src/plugins/data/common/search/expressions/eql_raw_response.test.ts

```diff
@@ -0,0 +1,133 @@
+    });
+  });
+
+  describe('converts sequences to table', () => {
+    test('simple docs response', () => {
+      const response: EqlRawResponse = {
+        type: 'eql_raw_response',
+        body: {
+          hits: {
+            sequences: [
+              {
+                events: [
+                  {
+                    _index: 'kibana_sample_data_ecommerce',
+                    _id: 'AncqUnMBMY_orZma2mZy',
+                    _score: 0,
+                    _source: {
+                      category: ["Men's Clothing"],
+                      currency: 'EUR',
+                      customer_first_name: 'Oliver',
+                      customer_full_name: 'Oliver Rios',
+                      customer_gender: 'MALE',
+                      products: [
+                        {
+                          base_price: 20.99,
+                          discount_percentage: 0,
+                        },
+                        {
+                          base_price: 24.99,
+                          discount_percentage: 0,
+                        },
+                      ],
+                      sku: ['ZO0417504175', 'ZO0535205352'],
+                      geoip: {
+                        country_iso_code: 'GB',
+                        location: {
+                          lon: -0.1,
+                          lat: 51.5,
+                        },
+                        continent_name: 'Europe',
+                      },
+                    },
+                  },
+                ],
+              },
+              {
+                events: [
+                  {
+                    _index: 'kibana_sample_data_ecommerce',
+                    _id: 'I3cqUnMBMY_orZma2mZy',
+                    _score: 0,
+                    _source: {
+                      category: ["Men's Clothing"],
+                      currency: 'EUR',
+                      customer_first_name: 'Boris',
+                      customer_full_name: 'Boris Bradley',
+                      customer_gender: 'MALE',
+                      products: [
+                        {
+                          base_price: 32.99,
+                          discount_percentage: 0,
+                        },
+                        {
+                          base_price: 28.99,
+                          discount_percentage: 0,
+                        },
+                      ],
+                      sku: ['ZO0112101121', 'ZO0530405304'],
+                      geoip: {
+                        country_iso_code: 'GB',
+                        location: {
+                          lon: -0.1,
+                          lat: 51.5,
+                        },
+                        continent_name: 'Europe',
+                      },
+                    },
+                  },
+                ],
+              },
+            ],
+          },
+        },
+      };
+      const result = eqlRawResponse.to!.datatable(response, {});
```

Comment:
```suggestion
      const result = eqlRawResponse.to?.datatable(response, {});
```

It will not crash with an error if `to` is missing, but it will fail the test.